adds/start.sh

```diff
@@ -54,11 +54,21 @@ configure_homeserver_yaml() {
 	mv ${ymltemp} "${filepath}"
 }
 
+configure_log_config() {
+        sed -i "s|.*filename:\s/homeserver.log|    filename: /data/homeserver.log|g" "/data/${SERVER_NAME}.log.config"
+}
+
```

Comment:
Arg - sorry I have misinterpreted the code you have added. You have do it right. Dont drop this part. I though the `configure_log_config` was executed on every start of matrix server

Sorry. My mistake @xsteadfastx